neqo-common/Cargo.toml

```diff
@@ -37,6 +37,7 @@ regex = { workspace = true }
 bench = ["test-fixture/bench"]
 build-fuzzing-corpus = ["hex"]
 ci = []
+test-fixture = []
```

Comment:
Wouldn't defining `test-fixture` as `optional = true` be enough?

https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

`cargo` should implicitly add the `test-fixture` feature.

Reply:
dev-dependencies are not allowed to be optional, I think?

Reply:
Ah, I didn't know.

https://github.com/rust-lang/cargo/issues/1596

Please ignore my comment then.